pkg/crc/machine/start.go

```diff
@@ -342,6 +342,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		return nil, errors.Wrap(err, "Error running post start")
 	}
 
+	if _, ok := os.LookupEnv("UNSUPPORTED_PODMAN_ONLY"); ok {
```

Comment:
Souldn't this before we start the DNS server inside the VM because this is only required for resolving openshift domain names.

Reply:
Yes not very good.. Lots of things should be turned off with this flag. I don't know if I should keep it as it is.
It is really convienent to have it for working with podman only..